tests/distributions/test_mvt.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+import torch
+from torch.distributions import StudentT
+
+from pyro.distributions import MultivariateStudentT
+from tests.common import assert_equal
+
+
+@pytest.mark.parametrize("batch_shape", [
+    (),
+    (3, 2),
+    (1,),
+], ids=str)
+# FIXME: what is a proper test for log_prob with dim > 1?
```

Comment:
one test is to make df very large and assert it's equal to multivariatenormal

Reply:
Yup, let me try to add that test.

Reply:
I tried it but probably df has to be very large for the convergence (in logprob scale) happen... so I replace it with a scaled mixture of normal test.